lte/protos/mconfig/mconfigs.proto

```diff
@@ -468,3 +468,23 @@ message GatewayDNSConfigRecordsItems {
   repeated string cname_record = 3;
   string domain = 4;
 }
+
+//----------------------------------------------------------------------------
+// accessd configs
+//----------------------------------------------------------------------------
+message AccessD {
+  enum LogLevel {
+    UNSET = 0;
+    DEBUG = 1;
+    INFO = 2;
+    WARN = 3;
+    ERROR = 4;
+  }
+  LogLevel log_level = 1;
+
+  string sctpd_downstream_service_target = 2;
+  string sctpd_upstream_service_target = 3;
+  string mme_sctpd_downstream_service_target = 4;
+  string mme_sctpd_upstream_service_target = 5;
+
+}
```

Comment:
we already have a service called `accessd` on orc8r. 

Is this service related to that one? If not, maybe it would be good to find a better name?

Reply:
what does accessd do in orc8r? we need a common name that terminates the procedures from (radio) access network rather than mme which is currently mme+amf+sgw-c. I think accessd is a good name to capture a converged access service.

Reply:
I could be wrong but I think accessd orc8r service is the access manager for operators (permissions, certificates) that are tied to a network. I'm not sure if having both `accessd` service names in both orc8r and agw scopes could lead to confusion, but I would advocate to having a different name for the AGW service.

Reply:
Yes `accessd` on the orc8r manages access control on Orc8r API access. (like which networks / gateways it has permision to read/write, etc.)